openquakeplatform/openquakeplatform/templates/data_viewer_base.html

```diff
@@ -29,8 +29,6 @@
     <link rel="stylesheet" href="{{ STATIC_URL }}css/d3.css" />
     <link rel="stylesheet" href="{{ STATIC_URL }}css/ng-table.min.css">
     <link rel="stylesheet" href="{{ STATIC_URL }}css/leaflet-dvf.css">
-    <link rel="stylesheet" href="{{ STATIC_URL }}bootstrap-extend/css/jasny-bootstrap.min.css">
```

Comment:
Are you sure this dependency is not used by any other application which inherits this template?